lambda/index.js

```diff
@@ -1,11 +1,23 @@
 const axios = require('axios');
+const jwt = require('jsonwebtoken');
+const env = require('dotenv');
+env.config();
 
 exports.handler = async (event) => {
+    const data = JSON.parse(event.body);
+
     const firebase_url = "https://us-central1-rcsls-2018.cloudfunctions.net/setWasherStatus"
```

Comment:
will be great to put this in env variable as well. generally a good idea so that the program is not dependent on this url